src/ui/components/statusrow/StatusRow.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Tooltip from '@material-ui/core/Tooltip';
+import BlockIcon from '@material-ui/icons/Block';
+import {EISIcon, RadioactiveWarningIcon} from 'eam-components/dist/ui/components/icons/index'
+
+const STATUS_KEYS = {
+    OUT_OF_SERVICE: "OUT_OF_SERVICE",
+    EIS: "EIS",
+    RADIOACTIVE: 'RADIOACTIVE',
+}
+
+const iconStyle = {
+    height: "25px",
+    width: "25px"
+}
+
+const STATUSES = [
+    {
+        key: STATUS_KEYS.OUT_OF_SERVICE,
+        shouldRender: (entity, entityType) => entity.outOfService === "true" || entity.outOfService === true,
```

Comment:
Is `entity.outOfService` `true` or `"true"` when enabled? Does it vary depending on the entity? If not, why have both checks?

Reply:
It varies, yes.